packages/storybook/src/nimble/text-field/text-field-matrix.stories.ts

```diff
@@ -492,6 +519,16 @@ export const colorThemeReadOnlyDisabledWithButtons: StoryFn = createFixedThemeSt
     colorThemeDarkGreenBackground
 );
 
+export const textFieldRequiredVisibleThemeMatrix: StoryFn = createMatrixThemeStory(
```

Comment:
The most recent storybook tests run is showing a lot of new text field stories in addition to just the one new one. Maybe we need to update the guid in preview.js to get it to re-evaluate correctly or something (also a change for the variable rename was jusst pushed so maybe that new story will be right)

Reply:
The 24 stories are still showing up as being new. I'm still trying to figure out why and how to fix it.